docs/ref/contrib/auth.txt

```diff
@@ -419,8 +419,7 @@ fields:
 
     .. attribute:: content_type
 
-        Required. A reference to the ``django_content_type`` database table,
-        which contains a record for each installed model.
+        Required. A foreign key to the :class:`~django.contrib.contenttypes.models.ContentType` model.
```

Comment:
Hello! Lines in the documentation should be wrapped at 80 characters: https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files

Reply:
fixed